src/common/KVStore/IKVStore.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef _IKVSTORE_
+#define _IKVSTORE_
+
+namespace snail{
+	
+class MockKVStore_result_type{
```

Comment:
1. These 2 classes should be in 2 files.
2. Could u plz tell me what this class means?
3. class name should not  includ`_`